lib/LLVMIRCodeGen/DebugInfo.cpp

```diff
@@ -160,18 +160,19 @@ LLVMIRGen::getOrCreateFunctionDebugInfo(llvm::Function *F, llvm::DIScope *scope,
     auto *DIFunctionTy = DIBuilder_->createSubroutineType(
         DIBuilder_->getOrCreateTypeArray(paramTys));
     // Create a debug information for the current function.
-#if LLVM_VERSION_MAJOR < 8 || FACEBOOK_INTERNAL
+#if LLVM_VERSION_MAJOR == 7
```

Comment:
Is it OK that FACEBOOK_INTERNAL was removed?

Reply:
fixing